python/py-h2/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-h2
-version             3.2.0
+version             4.0.0
```

Comment:
in this version support for PY27/PY34/PY35 has been removed (see the release [notes](http://python-hyper.org/projects/h2/en/stable/release-notes.html#api-changes-backward-incompatible)). 

So you'll need to deal with this in the Portfile and restrict these subports to their latest supported versions or, if they have no dependents, they possibly could be removed.